cmd/tailscale/cli/up.go

```diff
@@ -166,6 +198,10 @@ func runUp(ctx context.Context, args []string) error {
 		}
 	}
 
+	if err := validateHostname(upArgs.hostname); err != nil {
```

Comment:
won't this always fail in the default case with the empty string for the flag?

Reply:
What a blunder! Thanks, fixed.